docs/04-smart-contracts/13-testing.md

```diff
@@ -250,9 +256,9 @@ You get a lot of flexibility when you are defining your own mocked contract. You
 
 Different chains and hubs in the Cosmos ecosystem may have some variations on how migrations are done on their respective networks. This section will attempt to outline those.
 
-### Juno 
+### Juno
```

Comment:
Back then it made sense to give references to active cosmwasm projects. But now there are many chains so it makes most sense to not give references. @merdoyovski can you change this with another PR?